src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -57,7 +59,17 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthorization(options =>
             {
-                options.AddPolicy(Policy.CanAccessOrganisation, policy => policy.RequireClaim("Organisation"));
+                options.AddPolicy(Policy.CanAccessOrganisation, policy => policy.RequireClaim(ApplicationClaimTypes.Organisation));
```

Comment:
I can only see this getting bigger. It might be a good idea to create a Service Collection Extensions class and add an AddCustomAuthorization method. Make a judgement call on this one. Doesn't have to be as part of the task but definitely something we need to keep an eye on.

Reply:
I agree it's going to possibly get too big. It'd be nice if we had some sort of... dare I say it, PolicyBuilder (not really a builder, but something to create the policies with the appropriate claims in a nice fashion), to trim it down a bit. I wasn't really a fan of the service collection extensions, but if we can find ways to tidy up the start up class I'm all for it, if they end up being ServiceCollectionExtensions I'm not opposed to it

Reply:
Haha! PolicyBuilder maybe an option but tidy up definitely sounds good.